plugins/xpressive/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+INCLUDE(BuildPlugin)
+
+SET(CMAKE_CXX_FLAGS "-Dexprtk_disable_sc_andor -Dexprtk_disable_enhanced_features -Dexprtk_disable_break_continue -Dexprtk_disable_comments -Dexprtk_disable_string_capabilities -Dexprtk_disable_rtl_io_file -Dexprtk_disable_rtl_vecops ${WERROR_FLAGS}")
```

Comment:
@gnudles  which plaform are you building where you see that exception being thrown?